src/components/footer.js

```diff
@@ -1,5 +1,58 @@
+import {
+  Center,
+  Container,
+  Divider,
+  Flex,
+  Image,
+  Link,
+  Text,
+  Box,
+} from "@chakra-ui/react";
 import React from "react";
+import { Link as ReactLink } from "react-router-dom";
+import LinkGrid from "./footer-link-grid";
 
 export function Footer() {
-  return <h1> This is Footer</h1>;
+  return (
+    <>
+      <Center bg="#E2E8F0" w="100%" position="relative" bottom="0" minH="320px">
```

Comment:
Having the color defined here is fine, all though if you defined it as part of a theme, you won't have to chase down this style afterward.

Reply:
I don't know how to fix this, tho I understand what you mean.